packages/cli/test/integration/generators/model.integration.js

```diff
@@ -205,6 +205,51 @@ describe('lb4 model integration', () => {
       assert.fileContent(expectedModelFile, /\[prop: string\]: any;/);
     });
 
+    it('scaffolds model with generated flag set to true for id property', async () => {
+      const property = {
```

Comment:
I tried to test if the `generated` flag is added to the id property. But it doesn't generate the id property at all. Is there any way to test it out? Or we can skip the test for now? @raymondfeng  @bajtos

Reply:
Weird. I am ok to skip the test for now, unless @raymondfeng objects.

Reply:
As we don't have a good way to mock prompts for composed generators, I'm fine to skip the test for now.